### PR TITLE
KostiantynSokoliuk/HW5

### PR DESCRIPTION
**Pull request by dedokTV**
Completed 5th lesson homework

### CODE REVIEW

File: Sokoliuk/HW4/HW4_task_01.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+print("Program for americans (Temperature converter)\n")
+print("If you want to stop the program, type 'stop'\n")
+
```
Comment (kolyasalubov) [inline Sokoliuk/HW4/HW4_task_01.py:3]: good job

File: Sokoliuk/HW5/HW5_task_01.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import random
+
+list_of_numbers = []
+
+for i in range(0, 10):
+    list_of_numbers.append(random.randint(1, 100))
+    i += 1
+
+print(f"List initialized\n{list_of_numbers}\n")
+
```
Comment (kolyasalubov) [inline Sokoliuk/HW5/HW5_task_01.py:10]: good job

File: Sokoliuk/HW5/HW5_task_02_and_03.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+print("Hello, to start a program press one of the folowing:\n \
+1 to print Fibonaci numbers;\n \
+2 to calculate the factorial;\n \
+3 to exit a program.\n")
+
```
Comment (kolyasalubov) [inline Sokoliuk/HW5/HW5_task_02_and_03.py:5]: this code is hard to read
I advise you to split this code into three separate parts
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)



n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])

**Review by kolyasalubov — APPROVED** [review]

File: Sokoliuk/HW5/HW5_task_02_and_03.py (hunk 3 above)
Comment (kolyasalubov) [reply Sokoliuk/HW5/HW5_task_02_and_03.py:5]: and you need to resolve conflict

**Comment by dedokTV** [comment]
Fixed what you asked, I tried your code, was impressed how concise it can be, but still, used my variant

**Pull request merged**